railties/CHANGELOG.md

```diff
@@ -140,4 +140,8 @@
 
     *Alex Robbin*
 
+*   Rename `railties/bin` to `railties/exe` to match the new Bundler executables convention.
+
+    *Islam Wazery*
+
```

Comment:
@matthewd Thanks for telling me, I will update the PR accordingly, sorry for it :smiley: